packages/boot/test/bootstrapTests/supports.js

```diff
@@ -188,9 +188,7 @@ export const getNodeTestVaultsConfig = async (
   bundleDir = 'bundles',
   specifier = '@agoric/vm-config/decentral-itest-vaults-config.json',
 ) => {
-  const fullPath = await importMetaResolve(specifier, import.meta.url).then(
-    u => new URL(u).pathname,
```

Comment:
I’m enthusiastic about replacing `new URL(u).pathname` with `url.filePathToURL` everywhere it’s found, in order to eventually be portable to Windows! Thank you.